File: C/Harvard/CS50/Week1/Problem_Set_1/mario-more/mario.c
#include "libs/cs50.c"
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int row = 0; row < height; row++)
    {
        for (int spaces = 0; spaces < height - row - 1; spaces++)
        {
            printf(" ");
        }

        for (int column = 0; column <= row; column++)
        {
            printf("#");
        }

        printf("  ");

        for (int column2 = 0; column2 <= row; column2++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Python/Harvard/CS50/Week_6/Problem_Set_6/sentimental-mario-more/mario.py
from cs50 import get_int

height = 0
column = 0
column2 = 0
while height < 1 or height > 8:
    height = get_int("Height: ")

for row in range(height):
    for spaces in range(height - row - 1):
        print(" ", end="")

    while column <= row:
        print("#", end="")
        column += 1
    column = 0

    print("  ", end="")

    while column2 <= row:
        print("#", end="")
        column2 += 1
    column2 = 0

    print()
